src/components/structural/header/Header.js

```diff
@@ -550,6 +552,47 @@ class Header extends Component {
         this.setState({ snackOpen: false });
     }
 
+    onExport = (scene = undefined) => {
+        let param = "";
+        if(scene) {
+            param = `?id=${scene}`;
+        }
+
+        fetch(`/apiv1/scenes/export${param}`, {headers: {"x-access-token": this.props.user.uid}}).then(async (resp) => {
+            switch(resp.status) {
+                case 204:
+                    alert("There are no scenes saved for you to export!");
+                    break;
+                case 200:
+                    let data = await resp.json();
+                    saveAs(new Blob([JSON.stringify(data)]), "MYR-export.json", {
```

Comment:
We should run the import/export itself in the back end and just use the front end to pass files back and forth.

Reply:
OK, I will have to restructure the import system to do this (not that bad).

Export has to be that way, the actual creation of JSON is created on the backend, but when invoked from the frontend it does not give a "Save File" prompt (I think this is a problem with `fetch` skipping past the `Content-Disposition` header)